webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -418,7 +420,14 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
         doc.newPage();
     }
 
-    private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, ScrambleRequest scrambleRequest, String globalTitle, int index) throws DocumentException, IOException {
+    private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, ScrambleRequest scrambleRequest, String globalTitle, int index, Locale locale) throws DocumentException, IOException {
+        
+        // internationalization
+        Locale currentLocale;
+        ResourceBundle messages;
+        currentLocale = new Locale(locale.getLanguage(), locale.getCountry());
+        messages = ResourceBundle.getBundle("net.gnehzr.tnoodle.server.webscrambles.Internationalization.MessagesBundle", currentLocale);
```

Comment:
Could you pull this logic out into a `translate(key, locale)` method that we then use below (instead of the `messages.getString` stuff? (Also, it seems silly to declare the variables and then initialize them on the next lines.)

Reply:
I agree about the silliness of this declaration.

This translate(key, locale) can easily be added, but what's the point?